src/utils/profile.ts

```diff
@@ -19,5 +20,21 @@ export const updateProfileEnteredVenueIds = async (
   }
 };
 
+export const getUserDisplayStatus = (
+  user?: WithId<User>
+): RecentUserStatusType => {
+  if (user?.status) return RecentUserStatusType.busy;
+
+  return user ? RecentUserStatusType.online : RecentUserStatusType.offline;
+};
+
 export const isCompleteProfile = (profile: FirebaseReducer.Profile<User>) =>
   isTruthy(profile.partyName) && isTruthy(profile.pictureUrl);
+
+export const getUserCurrentLocation = (user?: WithId<User>) => {
+  if (!user?.lastSeenIn) {
+    return undefined;
+  }
+
+  return Object.keys(user.lastSeenIn)[0];
+};
```

Comment:
What's the intent of this?

Reply:
removed as unused. fixed in 5a3581bc4abfb14880e2e621a12c0994e01cace8